data/mods/Magiclysm/Spells/druid.json

```diff
@@ -443,5 +443,32 @@
     "damage_increment": 4,
     "//": "The spell can only be permanent without being at maximum level because of how items with charges work",
     "flags": [ "SOMATIC", "NO_LEGS", "PERMANENT" ]
+  },
+  {
+    "id": "cause_rot",
+    "type": "SPELL",
+    "name": "Cause Rot",
+    "description": "A spell from a darker sect of druidry, created to forcibly begin the circle of life anew.  For without rot, you cannot have growth.  Affects a huge area, but is indiscriminate;  even rusting machines into nothing but dust, and slow both in casting and effect.  Once lost entirely to the passage of time, this spell was thought to be no more than a myth, the stuff of fairy tales, albeit grim ones.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SOMATIC", "NO_PROJECTILE", "CONCENTRATE", "VERBAL" ],
+    "max_level": 30,
+    "spell_class": "DRUID",
+    "energy_source": "MANA",
+    "difficulty": 8,
+    "damage_type": "pure",
+    "base_casting_time": 10000,
```

Comment:
don't you think 100 seconds is a bit long for any damaging spell? i know it has a really big area, but it also costs 1k mana which is quite a large chunk.

Reply:
added casting_time_increment to cover it, so higher level would be a minor 5 second sniper spell